autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -73,6 +73,47 @@ pub struct BigTableClientImpl {
     pool: BigTablePool,
 }
 
+fn timestamp_filter() -> Result<data::RowFilter, error::BigTableError> {
+    let mut timestamp_filter = data::RowFilter::default();
+    let bt_now: i64 = SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .map_err(error::BigTableError::WriteTime)?
+        .as_millis() as i64;
+    let mut range_filter = data::TimestampRange::default();
+    range_filter.set_start_timestamp_micros(bt_now * 1000);
+    timestamp_filter.set_timestamp_range_filter(range_filter);
+
+    Ok(timestamp_filter)
+}
+
+/*
+/// Filter based on the calculated expiry of the data row.
+/// NOTE: Filtering on a specific cell value will return only that
+/// cell value in the row.
+fn expiry_filter() -> Result<data::RowFilter, error::BigTableError> {
```

Comment:
Do we want to keep this?

Reply:
I was going to make an argument that we should keep the comment in lieu of a document that includes the various "Bigtable Learnings" we've had to make (because Bigtable is weird), but then I realized I should just create that document.